README.md

```diff
@@ -57,6 +57,20 @@ You can mark certain CMS users as developers in your site's config, so they can
 
 ## Customisation
 
+BetterNavigator can be made translucent by adding the following config setting:
```

Comment:
Edit: can be made translucent **when collapsed** by adding

Reply:
Sure thing. This is done.